lib/adapter_base.coffee

```diff
@@ -94,10 +95,10 @@ class Adapter
    * @return {Promise}
   ###
   renderFile: (file, opts = {}) ->
-    opts = _.clone(opts, true)
+    opts = clone(opts, true)
     (new File(file))
       .read(encoding: 'utf8')
-      .then _.partialRight(@render, _.extend({ filename: file }, opts)).bind(@)
+      .then partialRight(@render, Object.assign({ filename: file }, opts)).bind(@)
```

Comment:
It was, but this is the first clear instance in the code that guarantees breakage 💥